src/components/playButton/PlayButton.tsx

```diff
@@ -0,0 +1,18 @@
+import React from 'react'
+import { Icon } from '../icon'
+import { space } from '../../theme/space'
+import { PlayButtonProps } from './PlayButton.types'
+import { ButtonWrapper } from './PlayButton.styles'
+
+export const PlayButton: React.FC<PlayButtonProps> = () => {
+  return (
+    <ButtonWrapper onClick={() => alert('clicked')}>
```

Comment:
Now the last part of the `button` update is to just perform a pass-through of the props from the parameter. This means that the component can be utilised as a button and receive any props a standard `button` element may receive.

This means that the explicit `onClick` can instead be changed to a spread of props (ie `{...props}`) which are taken into the component on the line above.

Reply:
This will lead to the `button` receiving the `onClick` implicitly from `props`. This is a safe operation since we know that the new type contains the `onClick` field.